xs/src/libslic3r/PerimeterGenerator.cpp

```diff
@@ -6,6 +6,23 @@
 
 namespace Slic3r {
 
+int
+PerimeterGenerator::num_loops(int perimeters, int extra_perimeters)
+{
+    int generated_perimeters = perimeters + extra_perimeters - 1;
+    long min_shell_thickness = scale_(this->config->min_shell_thickness);
```

Comment:
or `auto` (which is my preference as you are instantiating the value already). If @alexrj disagrees, go with what he said. ;)